catalog/transformer/catalog-transformer-geojson-input/src/test/java/ddf/catalog/transformer/input/geojson/GeoJsonExtensibleTest.java

```diff
@@ -481,6 +485,80 @@ public void testBasicMetacardTypeNoMetacardType()
     verifyBasics(metacard);
   }
 
+  @Test
+  public void testTransformWithoutIdParam() throws IOException, CatalogTransformerException {
+    Metacard metacard =
+        transformer.transform(new ByteArrayInputStream(sampleBasicMetacard().getBytes()));
+
+    assertThat("ddf.metacard", is(metacard.getMetacardType().getName()));
```

Comment:
Might be good to set these strings as variables, since they are getting used so often across this class

Reply:
Didn't realize it before, but it looks like there are some constants defined at the top of the file already!